docs/edgeql/functions/math.rst

```diff
@@ -0,0 +1,139 @@
+.. _ref_eql_functions_math:
+
+
+====
+Math
+====
+
+In general the mathematical functions are designed to keep the
+separation between :eql:type:`decimal` values and the rest of the
+:ref:`numeric types <ref_datamodel_scalars_numeric>`. The philosophy
+is that using :eql:type:`decimal` should be very explicitly
+opt-in, but once used the values should not be accidentally cast
+into a numeric type with less precision.
```

Comment:
I'm not sure this paragraph is relevant to the contents of this section.  I'd move it to the description of the `decimal` type.

Reply:
OK, I'll move it to the numeric types section.